roles/satosa/tasks/main.yml

```diff
@@ -193,21 +193,6 @@
         dest: "{{ satosa_env_dir }}/satosa.service"
         mode: 0644
 
-    - name: Create mapper directory
-      file: path="{{ satosa_env_dir }}/mapper" state=directory mode=0755
```

Comment:
Could you replace this by a file: state=absent so that existing directories will get removed?

Reply:
I will create a fresh venv on deployment anyway, so this point is moot.